functional/provision/setup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e
```

Comment:
ok, one might (for whatever reason) want to do other stuff in setup.sh, though, so I'll make setup.sh call a new install_go.sh